google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardDeployTask.java

```diff
@@ -63,6 +65,16 @@ public void execute(ProcessStartListener startListener) {
         .withLabel(isFlexCompat ? "flex-compat" : "standard")
         .ping();
 
+    if (CloudSdkService.getInstance().validateCloudSdk().contains(
+        CloudSdkValidationResult.CLOUD_SDK_NOT_FOUND)) {
```

Comment:
Is `CloudSdkValidationResult.MALFORMED_PATH` something we should check for here as well?